sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx

```diff
@@ -24,6 +24,64 @@ import Amount from '~/react/components/helpers/Amount';
 import plansStyles from './plans.module.css';
 import Skeleton from 'react-loading-skeleton';
 
+interface PricingColumnHeaderProps {
+  plan: PlanIntervalPricing;
+  selectedInterval: IntervalKeys;
+}
+
+const PricingColumnHeader = ({
+  plan,
+  selectedInterval
+}: PricingColumnHeaderProps) => {
+  const { config } = useFrontier();
+  const selectedIntervalPricing = plan.intervals[selectedInterval];
+  const showPerMonthPrice =
+    config?.billing?.showPerMonthPrice === true && selectedInterval === 'year';
+
+  const perIntervalLabel = showPerMonthPrice
+    ? 'per seat/month'
+    : `per seat/${selectedInterval}`;
+
+  const amount = showPerMonthPrice
+    ? selectedIntervalPricing.amount / 12
+    : selectedIntervalPricing.amount;
+
+  const actualPerMonthAmount = plan.intervals['month']?.amount || 0;
+  const showDiscount = showPerMonthPrice && actualPerMonthAmount > 0;
```

Comment:
What does the second part here check for `actualPerMonthAmount > 0`?

Reply:
it is needed to calculate discount, as we need to divide the price delta with `actualPerMonthAmount`. thus checking if value is greater than 0.
i have refactored to make it more readable